src/components/interactive-map/interactive-map.tsx

```diff
@@ -64,7 +69,7 @@ const InteractiveMap: React.FC<InteractiveMapProps> = (props) => {
       return color.replace(
         /rgb(a?)\((\d+), (\d+), (\d+)(, [\d.]+)?\)/,
         // "rgba($2, $3, $4, 0.4)"
-        `rgba($2, $3, $4, ${isSelected ? 0.3 : isHovered ? 0.4 : 0.5})`
+        `rgba($2, $3, $4, ${isSelected ? 0.3 : isHovered ? 0.4 : 0.5})`,
```

Comment:
Tiny thing, and we can pick this up in a future iteration, but this `0.3`, `0.4`, and `0.5` should be extracted into named constants like `PROVINCE_COLOR_OPACITY_HOVERED`.